drivers/filesystems/ext2/src/devctl.c

```diff
@@ -511,7 +527,11 @@ Ext2ProcessVolumeProperty(
             RtlCopyMemory(Property2->UUID, Vcb->SuperBlock->s_uuid, 16);
             Property2->DrvLetter = Vcb->DrvLetter;
 
+#ifdef __REACTOS__
             if (Property2->bHidingPrefix == Vcb->bHidingPrefix) {
+#else
+            if (Property2->bHidingPrefix = Vcb->bHidingPrefix) {
```

Comment:
Is that a bug in the original code, that we fixed in ReadctOS? Or is the original code correct here, and we just tried to mute some static code check and did it wrong?

Reply:
it's a bug in the original code that we fixed indeed. After this PR, I'll try to upstream those fixes. but unfortunately, as stated in the Jira ticket, upstream seems to be abandoned. only https://github.com/bobranten/Ext4Fsd seems to still be around

Reply:
It would be worth looking at whether the ext4fsd of Bo Branten also covers ext2 and 3, and if so, how much does the code change wrt. that ext2/3 driver we have. In case of positive observations, maybe one could think about updating to that ext4fsd ?

Reply:
Yes, I'll try to have a look into it :)

Reply:
> it's a bug in the original code that we fixed indeed. After this PR, I'll try to upstream those fixes. but unfortunately, as stated in the Jira ticket, upstream seems to be abandoned. only https://github.com/bobranten/Ext4Fsd seems to still be around

This also have this bug, so it's worth to send a PR. See https://github.com/bobranten/Ext4Fsd/blob/d9da413ad28afa3c0519d730da60f8598bbaea32/Ext4Fsd/devctl.c#L514

Reply:
after this PR is merged, I'll open a PR on ext4fsd with ReactOS fixes. Then I'll see to use Ext4fsd in ROS :)